.github/workflows/frontier_amd/build.sh

```
@@ -15,12 +15,38 @@ fi
-    done
-else
-    ./mfc.sh test -a --dry-run -j 8 $build_opts
-fi
+max_attempts=3
+attempt=1
+while [ $attempt -le $max_attempts ]; do
+    echo "Build attempt $attempt of $max_attempts..."
+    if [ "$run_bench" == "bench" ]; then
+        build_cmd_ok=true
+        for dir in benchmarks/*/; do
+            dirname=$(basename "$dir")
```

Shellcheck (0.11.0)

[warning] dirname appears unused. Verify use (or export if used externally).

(SC2034)